### ISSUE TITLE
Questions regarding the outputs of the codes

### ISSUE THREAD

**Issue by jingliu-ut** [fenghuibao/dSort-Seq#2]
I ran your example files and got the `lamb.pickle`, `mu.pickle`, and `sigma.pickle`. According to my understanding of your paper, you would get the estimated mean of each individual variant from Equation 1 in your methods section.
![image](https://github.com/user-attachments/assets/53c07056-3b01-4650-985b-01321257fddd)

Is my understanding correct? If so, could you provide the code for this part?

Thanks!

**Comment by fenghuibao** [comment]
Hi Jingliu,

The equation you mentioned is the log-scaled fluorescence distribution. To calculate the mean and SD you need to use Eq. 11 and Eq. 12 (see Obtaining expression characteristics from cytometry data through LGMM fitting
 in Method). 

Best,
Huibao

**Comment by jingliu-ut** [comment]
Thank you for responding, Huibao! Just to clarify things, to get the estimated Mean and SD for each variant (Fig 2I & J), I would use the outputs from your script (`lamb.pickle`, `mu.pickle`, and `sigma.pickle`) and plug them into equations 11 and 12?

**Comment by fenghuibao** [comment]
Yes, exactly!

Best,
Huibao

**Comment by jingliu-ut** [comment]
Thanks!

**Comment by jingliu-ut** [comment]
Hi Huibao,

I have a question regarding lamb_i in equations 11 and 12. The `lamb.pickles` output file has two columns, which I assume represent the weights of the first and second Gaussian components. How did you calculate lamb_i from lamb1_i and lamb2_i? Also, why are there negative values in these columns? I noticed that these negative values exist in both my output files and the output files from your example dataset. I thought the lamb values were supposed to be non-negative and add up to 1.

Thanks,
Jing

**Comment by fenghuibao** [comment]
Hi Jing,

I'm sorry for not getting back to you sooner. Yes, the lamb.pickle file outputs the weights, but they are in the form of logits rather than probits. You’ll need to apply the softmax function to convert them into probabilities.

For your reference, I’ve uploaded code that demonstrates the calculation of the Mean and SD:

https://github.com/fenghuibao/dSort-Seq/blob/main/code/example/cal_Mean_SD.py

Best,
Huibao

**Comment by jingliu-ut** [comment]
Thank you for confirming!

Jing
